main.go

```diff
@@ -26,6 +26,7 @@ type config struct {
 	DepWarnning  string
 	Script       []string `yaml:"script"`
 	Build        []string `yaml:"build"`
+	Runners      []string `yaml:"runners"`
```

Comment:
I was a bit swayed from seeing `build` vs `runners`. Maybe we can come up with a better name instead of `runners`? Though for now this will suffice.